### PR TITLE
Transfer: display estimated transaction fee, requires #6302

### PR DESCRIPTION
**Pull request by xiphon**
monero-gui-estimate-fee

### CODE REVIEW

**Comment by SamsungGalaxyPlayer** [comment]
Should we also show the USD or desired fiat current conversion for the fee?

**Comment by xiphon** [comment]
> Should we also show the USD or desired fiat current conversion for the fee?

Updated.

monero-gui-estimate-fee-usd

**Comment by rating89us** [comment]
When estimated fee is less than 0.01 USD, we should display it as (<0.01 USD), instead of (~0.00 USD).

> > Should we also show the USD or desired fiat current conversion for the fee?
> 
> Updated.
> 
> monero-gui-estimate-fee-usd

**Comment by selsta** [comment]
I guess this will stay open for a while until we update to libwallet_api to master?

**Comment by xiphon** [comment]
> When estimated fee is less than 0.01 USD, we should display it as (<0.01 USD), instead of (~0.00 USD).

Fixed.

> I guess this will stay open for a while until we update to libwallet_api to master?

Yep.

**Comment by rating89us** [comment]
We should remove the extra zeros at the end of the estimated fee:
`0.000023440000 XMR` should be `0.00002344 XMR`

**Comment by xiphon** [comment]
> remove the extra zeros at the end of the estimated fee

Updated.

File: js/Utils.js [hunk 1]
```diff
@@ -133,3 +133,7 @@ function capitalize(s){
     if (typeof s !== 'string') return ''
     return s.charAt(0).toUpperCase() + s.slice(1)
 }
+
+function removeTrailingZeros(value) {
+    return (value + '').replace(/(\.\d*[1-9])0+/, '$1');
```
Comment (rating89us) [inline js/Utils.js]:
```suggestion
    var lastchar = value.charAt(value.length-1);
    if (lastchar == "0") {
        return (value + '').replace(/(\.\d*[1-9])0+/, '$1');
    } else {
        return value;
    }
```
I'm trying to use this function in the amount value of tx confirmation popup (which is still WIP).
When the amount is 0.000101010101, it returns 0.00010101011. So it should run only when the last digit is 0.

Comment (xiphon) [reply js/Utils.js]: Fixed

**Pull request merged**